BUILD.bazel

```diff
@@ -9,3 +12,50 @@ filegroup(
     name = "node_modules_none",
     srcs = [],
 )
+
+# The entire content of the real WORKSPACE file in this directory is only meant for local development.
+# So we generate a trivial one for the distribution.
+genrule(
+    name = "generated_WORKSPACE",
+    srcs = [],
+    outs = ["WORKSPACE"],
+    cmd = "echo \"workspace(name = \\\"build_bazel_rules_nodejs\\\")\" > $@",
+)
+
+npm_package(
+    name = "rules_nodejs_package",
+    srcs = [
+        "BUILD.bazel",
+        "LICENSE",
+        "defs.bzl",
+        "package.bzl",
+        "//third_party/github.com/gjtorikian/isBinaryFile:LICENSE",
```

Comment:
Wouldn't the LICENSE file be included in the `//third_party/github.com/gjtorikian/isBinaryFile` dep?

Reply:
no, that dep is just a js_library or ts_library

but I'll add a package_contents there so we don't have two references into that directory